python/plugins/processing/core/Processing.py

```diff
@@ -201,9 +201,9 @@ def loadAlgorithms():
         Processing.updateProviders()
         providers = [p for p in Processing.providers if p.getName() != "model"]
         for provider in providers:
-            providerAlgs = provider.algs
+            provider_algs = provider.algs
```

Comment:
hm, also a PEP8 suggestion.  But I'd ignore it, because our C++ code and hence the API doesn't follow this either.